modules/apps/frontend-js/frontend-js-components-web/src/main/java/com/liferay/frontend/js/components/web/internal/servlet/taglib/FrontendComponentsTopHeadDynamicInclude.java

```diff
@@ -74,6 +84,26 @@ protected void activate(Map<String, Object> properties) {
 				FFFrontendJSComponentsConfiguration.class, properties);
 	}
 
+	private void _createRuntimeFeatureFlag(
+		StringBundler sb, String featureFlagName, boolean validator) {
+
+		sb.append("Liferay.__FF__.");
+		sb.append(featureFlagName);
+		sb.append(" = ");
+		sb.append(validator);
+		sb.append(StringPool.SEMICOLON);
+	}
+
+	private boolean _shouldEnableCustomDialogs() {
+		if (GetterUtil.getBoolean(
+				PropsUtil.get("feature.flag.enableCustomDialogs"))) {
+
+			return true;
+		}
+
+		return false;
+	}
```

Comment:
We can simplify this by adding the second argument to GetterUtil, that represents the default value
```java
GetterUtil.getBoolean(PropsUtil.get("feature.flag.enableCustomDialogs"), false);
```

Reply:
My suggestion is also changing the logic a bit. In your solution, if a user sets `feature.flag.enableCustomDialogs=false`, it would enable FF.

Reply:
We cannot pass false as a default value here. See https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/getter_util_check.markdown

Reply:
The weirdest thing here is because SF started to throw this error today for me.

Reply:
> We cannot pass false as a default value here. See https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/getter_util_check.markdown

Interesting! But, IIUC, that's just SF, saying that default value of a boolean getter should [always be set to false](https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/GetterUtil.java#L649-L651). `_shouldEnableCustomDialogs` still feels very unnecessary, duplicating the job of `GetterUtil`.

Reply:
Yes. One thing we may do about it is communicating with Brian about removing the default value of GetterUtil?